grakn-core/src/main/java/ai/grakn/graql/ComputeQuery.java

```diff
@@ -51,7 +51,7 @@
      * @return a ComputeQuery with the subLabels set
      */
     @CheckReturnValue
-    ComputeQuery<T> in(Collection<Label> subLabels);
+    ComputeQuery<T> in(Collection<? extends Label> subLabels);
```

Comment:
Why does this need the wildcard now?

Reply:
A fun sequence of reasons:
- I added `ComputeQuery#subLabels`
- I gave `subLabels` the most generic type (`Collection<? extends Label>`) to protect us from breaking changes etc.
- At some point, code goes `query2.in(query1.subLabels())`, so now `in` must also accept the most generic type.